vuecomponents/bundle/src/main/bundle/tag-selector_eu.properties

```diff
@@ -0,0 +1,8 @@
+search_or_add=Buscar o añadir etiqueta
+search_filter=Buscar usando etiquetas
+add_new=Añadir texto como nueva etiqueta
+no_options=No hay etiquetas para el sitio
+no_results=No se han encontrado resultados para la búsqueda actual
+select=Pulsar Enter para seleccionar etiqueta
+selected=Seleccionada
+deselect=Deseleccionar
```

Comment:
@PabloSanRoman would you mind providing translations for EU?

Reply:
Hi @bgarciaentornos , here it is!

search_or_add=Bilatu edo erantsi etiketa
search_filter=Bilaketa egin etiketak erabiliz
add_new=Gehitu testua etiketa berria gisa
no_options=Etiketarik ez dago gunerako
no_results= Ez da emaitzarik aurkitu bilaketarako
select=Sakatu Enter etiketa aukeratzeko
selected=Hautatua
deselect=Deshautatu